src/models/EventPayload.ts

```diff
@@ -78,4 +80,13 @@ export interface EventPayload {
     /** The algorithm to use to anonymize the IP address after collection. */
     algorithm?: string;
   };
+  /**
+   * Information used to associate analytics with a particular user.
+   *
+   * @remarks
+   * Keys are visitor methods and values are visitor IDs
```

Comment:
We could also add the regex for a visitor method `^[a-zA-Z0-9_-]{1,16}$ ` and visitor ID `^[\x20-\x7E]{1,64}$`